### ISSUE TITLE
Add a button to pause all animations.

### ISSUE THREAD

**Issue by jasonwebb** [jasonwebb/supertunnel#1]
Having an animation that starts automatically on page load and lasts more than five seconds is a violation of [WCAG 2.2.2](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html), and can be fixed easily with a clear mechanism (like a button) to pause or stop it.

- [x] Add a new button to the `<nav>` region that toggles between two states (pause and play) when activated.
- [x] Use the `aria-pressed` attribute to convey to screen readers the state of the button.
- [x] Only rotate the globe when the pause button in the "play" state.
- [x] Disable the pulsing CSS animation on the draggable handle when animations are paused.
